### ISSUE TITLE
Evaluating embeddings

### ISSUE THREAD

**Issue by julianbernauer** [julianbernauer/QTA#4]
For instance by predicting new keywords?

**Comment by fedenanni** [comment]
We should consider two types of evaluation:
a) when using general pre-trained embeddings to show that they capture relevant properties in the pol-sci domain
b) when using our trained embeddings that they capture both pol-sci relevant properties and general linguistic properties

**Comment by julianbernauer** [comment]
Good point. Having both type of trained data would surely make use more immune to critique.

**Comment by julianbernauer** [comment]
Something for construct validation: qualitative assessment of high-scoring sentences
